examples/nextjs-with-typescript/src/app/react-wrappers/player.tsx

```diff
@@ -0,0 +1,83 @@
+import {
```

Comment:
Minor refactor of our original example app. Note that this new version is a fully server-side rendered component with the new Next 13 component architecture.

TODO: Re-add toggle button to mount/dismount the controller somewhere (which cannot be defined as an RSC) or add an additional page for this to ensure we can still easily smoke test for e.g. GC concerns.